### DISCUSSION TITLE
Why typescript warning

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#4933] [Q&A]
how to fix warning. i'm use in nextjs.13.5
![Screenshot 2566-10-01 at 14 20 12](https://github.com/mantinedev/mantine/assets/12431734/eb8ecb33-2c40-40e3-bdc8-f56b6719bf05)

And tsconfig.json
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "bundler",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "plugins": [
      {
        "name": "next"
      }
    ],
    "paths": {
      "@/*": [
        "./*"
      ]
    }
  },
  "include": [
    "next-env.d.ts",
    "**/*.ts",
    "**/*.tsx",
    ".next/types/**/*.ts"
  ],
  "exclude": [
    "node_modules"
  ]
}

**Comment by rtivital** [comment]
It is not a typescript warning. To set up editor for postcss install postcss extension for VSCode

**Comment by goodlythink** [comment]
Nice!. thank you very much
